dpgen/data/arginfo.py

```diff
@@ -35,6 +36,135 @@ def init_reaction_mdata_arginfo() -> Argument:
+        Argument("skip_relax", bool, optional=False, doc=doc_skip_relax),
+        Argument("pert_numb", int, optional=False, doc=doc_pert_numb),
+        Argument("pert_box", float, optional=False, doc=doc_pert_box),
+        Argument("pert_atom", float, optional=False, doc=doc_pert_atom),
+        Argument("md_nstep", int, optional=False, doc=doc_md_nstep),
+        Argument("coll_ndata", int, optional=False, doc=doc_coll_ndata),
+        Argument("type_map", list, optional=True, doc=doc_type_map),
+    ], sub_variants=init_bulk_variant_type_args(),
+    doc=doc_init_bulk) 
+
+def init_surf_jdata_arginfo() -> Argument:
+    """Generate arginfo for dpgen init_surf jdata.
+    
+    Returns
+    -------
+    Argument
+        dpgen init_surf jdata arginfo
+    """
+    doc_init_surf = "Generate initial data for surface systems."
+    doc_stages = "Stages for `init_surf`."
+    doc_elements = "Atom types."
+    doc_potcars = "Path of POTCAR."
+    doc_cell_type = "Specifying which typical structure to be generated. **Options** include fcc, hcp, bcc, sc, diamond."
+    doc_super_cell = "Size of supercell."
+    doc_from_poscar = "Deciding whether to use a given poscar as the beginning of relaxation. If it's true, keys (`cell_type`, `latt`) will be aborted. Otherwise, these two keys are **necessary**."
+    doc_from_poscar_path = "Path of POSCAR for VASP or STRU for ABACUS. **Necessary** if `from_poscar` is true."
+    doc_latt = "Lattice constant for single cell."
+    doc_layer_numb = "Number of equivalent layers of slab."
+    doc_z_min = "Thickness of slab without vacuum (Angstrom). If `layer_numb` is set, `z_min` will be ignored."
+    doc_vacuum_max = "Maximal thickness of vacuum (Angstrom)."
+    doc_vacuum_min = "Minimal thickness of vacuum (Angstrom). Default value is 2 times atomic radius."
+    doc_vacuum_resol = "Interval of thickness of vacuum. If size of `vacuum_resol` is 1, the interval is fixed to its value. If size of `vacuum_resol` is 2, the interval is `vacuum_resol[0]` before `mid_point`, otherwise `vacuum_resol[1]` after `mid_point`."
+    doc_vacuum_numb = "The total number of vacuum layers **Necessary** if vacuum_resol is empty."
+    doc_mid_point = "The mid point of head region and tail region. **Necessary** if the size of vacuum_resol is 2 or 0."
+    doc_head_ratio  = "Ratio of vacuum layers in the nearby region with denser intervals(head region). **Necessary** if vacuum_resol is empty."
+    doc_millers = "Miller indices."
+    doc_relax_incar = "Path of INCAR for relaxation in VASP. **Necessary** if `stages` include 1."
+    doc_scale = "Scales for transforming cells."
+    doc_skip_relax = "If it's true, you may directly run stage 2 (perturb and scale) using an unrelaxed POSCAR."
+    doc_pert_numb = "Number of perturbations for each POSCAR."
+    doc_pert_box = "Percentage of Perturbation for cells."
+    doc_pert_atom = "Perturbation of each atoms (Angstrom)."
```

Comment:
Perturbation of atom coordinates (Angstrom). Random perturbations are performed on
three coordinates of each atom by adding values randomly sampled from a uniform distribution in the range [−pert_atom, pert_atom].